code/modules/mob/living/silicon/robot/inventory.dm

```diff
@@ -38,6 +32,12 @@
 	else if(held_items[3] == O)
 		inv3.icon_state = "inv3"
 		held_items[3] = null
+
+	O.forceMove(module) //Return item to module so it appears in its contents, so it can be taken out again.
+
+	if(O.flags_1 & DROPDEL_1)

```

Comment:
This should either be moved before `forceMove` or removed, as the item has already been deleted as a result of `forceMove` calling `dropped()`. As this behavior is in current master and borgs have not reported losing modules, it should be removed as completely superfluous admeme-check. Admins can remove DROPDEL if they add modules via VV to borgs.

Reply:
It's useful for admins so I'm going to leave it in for now, but I've moved it to a location where it will actually do something.